api/api/src/test/scala/uk/ac/wellcome/platform/api/QueryParamsTest.scala

```diff
@@ -0,0 +1,5 @@
+package uk.ac.wellcome.platform.api
+
+import org.scalatest.FunSuite
+
+class QueryParamsTest extends FunSuite {}
```

Comment:
Were you planning to add any tests here?

Reply:
Nope, I was going to but we had similar tests elsewhere, I'dlike to have a strategy for what, where and why we tests things as there's a few different bits all over the place.

Reply:
I think it is just that the compiler infers that you mean the case object rather than the class, exposes a function taking an `Int`.


Interestingly you can get the compiler into a tangle like so:
```
case class IntTaker(i: Int)
case object IntTaker {
  def apply(i: Int): IntTaker = IntTaker(i)
}
val intMaker = List(1,2,3)


intMaker map IntTaker.apply
```

Reply:
Yup I understood that's what was going on, just posted cos caught me out at first :)